libs/novui/src/json-schema-components/widgets/Variable.tsx

```diff
@@ -86,5 +86,5 @@ function getValidatedVariable(text: string = '', possibleVariables: Set<string>)
     return;
   }
 
-  return variableName;
+  return variableContent;
```

Comment:
Fix for [this](https://github.com/novuhq/novu/pull/6330#discussion_r1730307006) . So liquid filters are applied to the rendered content